model/flow/chunk.go

```diff
@@ -111,3 +116,8 @@ func (cl ChunkList) ByIndex(i uint64) (*Chunk, bool) {
 func (cl ChunkList) Len() int {
 	return len(cl)
 }
+
+type BlockExecutionDataRoot struct {
```

Comment:
This is already defined in module/executiondatasync/execution_data/execution_data.go, why do we redefine it here?

Reply:
I think there was a import cycle, as this was using things from flow/model